src/functions/help/main.py

```diff
@@ -0,0 +1,32 @@
+from dbm import _Database
+from typing import Any, Optional, Tuple
+
+import functions_framework
+from google.cloud.firestore_v1.collection import CollectionReference
+from google.cloud.firestore_v1.document import DocumentReference
+from google.cloud.firestore_v1.client import Client
+from google.cloud.firestore_v1.base_document import DocumentSnapshot
+
+@functions_framework.http
+def hello(request) -> Tuple[str, int]:
+    """Return the requested list of all actived function"""
+
+    request_json: Optional[Any] = request.get_json(silent=True)
+
+    if request_json:
+        server_id = request_json.get("server_id", None)
+        
+        database: Client = Client(project="archy-f06ed")
+        
+        function_collection: CollectionReference = database.collection("servers").document(server_id).collection("function")
+        doc_ref = function_collection.collection("functions")
+        docs: DocumentSnapshot = doc_ref.where(u'active',u'==', True ).stream()
+        
+        if not server_id:
+            print("Exit: Missing data")
+            return "", 200
+        
+        if docs.exists:
+            return list(docs), 200
```

Comment:
`return list(docs), 200` -> `return [doc.get("description") for doc in docs]`
but we want a string of description and not just a list so we need to str concat the return

Reply:
done

Reply:
well yeah but what do we want this command to return ?
We want something nice for the user to understand how to use the commands

ex:
```bash
!help
>
!hello -> Greet the user back
!froge -> Send a random froge emoji selected from the server
!level -> Return the user's level
!level @username -> Return the selected user's level
... 
```

Something like this for a starter would be nice, we could improve the layout and design later

right now we are returning:

```bash
[ "Greet the user back","Send a random froge emoji selected from the server"...] 
```

Reply:
Alright! the change will should match the example 
id -> function name
description -> description of the function